1_slices_modules_and_objects/exercises/generic_ordered_vec/src/lib.rs

```diff
@@ -118,6 +123,7 @@ mod tests {
 
         assert!(ov.is_sorted());
 
+        assert_eq!(ov.pop(), 0.00000005);
```

Comment:
Good catch! Can you please open an issue on my repository (or even a pull request with just that change) so that we remember to fix it? Thanks!

Reply:
sure

Reply:
Thanks!

Reply:
I tried to open a PR for this but I receive a bad access right issue so i have created an issue instead